### ISSUE TITLE
Logo showed in presentation is not consistent with SSW

### ISSUE THREAD

**Issue by tiagov8** [wicksipedia/PointBank#11]
...about the logo, the concept is OK...however...
 
I think the font is not ideal for a web app. It's also pretty inconsistent with everything we have at SSW. The colors are OK, but I'd use a darker grey instead of black. The hand icon says a lot, it's too complex.

_" if you can’t explain it to a six year old, then you don’t understand it yourself. "_
Albert Einstein

It also makes it hard to apply it in small sizes and still be "readable" as a favicon for example.
 
So I wouldn't give it a pass.
 
Here are some links to help making a pontential PointBank logo:
	
- [https://uhurunetwork.com/branding-design/](https://uhurunetwork.com/branding-design/)
- [https://www.ssw.com.au/ssw/company/Web-Reference.aspx#Colors](https://www.ssw.com.au/ssw/company/Web-Reference.aspx#Colors)

**Comment by calumjs** [comment]
Agreed, was generated in 5 minutes using an online free logo maker because I needed one on that slide.
Someone could create a proper logo and add it to the repo with a pull request if required.
